### DISCUSSION TITLE
错误更正与修改建议

### DISCUSSION THREAD

**Discussion by fan-ziqi** [homalozoa/ros2_for_beginners_code#4] [General]
## 错误更正与修改建议

1. 换个英文字体
2. 冒号 `:` 斜杠 `/` 等在代码中或链接中的格式问题
3. 有些符号如 `$` 、 `-` 前后文不统一的问题
4. P7最后一句 ~~厘清~~ 理清思路
5. P12-1.1.4 第二段倒数第二行应为 `~/.bashrc` 
6. P13-1.2 第二段 ~~rclC++~~ rclcpp
7. P14 代码1-7多了俩空格（相同问题还有P23页代码1-17、P41、后面太多了不列了）
8. P98 std::endl或“\n”删除空格
9. P100 指令之间加空格
10. P313-8.1 第二段第二行码垛 ~~拌匀~~ 搬运
11. P315第四行MIT后加空格
------

修改于 2022.11.26

**Comment by homalozoa** [comment]
感谢反馈！

**Reply by 15906185391** [reply]
p74 图2-2 最上面的onConfigure:漏了e

**Comment by homalozoa** [comment]
9 by design @fan-ziqi 

9的话指的是基于ros2run、ros2launch等功能包实现的指令功能，并非是纯粹的ros2 run和ros2 launch的指令。
